###############################################################################
#
# Copyright (c) 2009, Oracle and/or its affiliates. All rights reserved.
#
# Permission is hereby granted, free of charge, to any person obtaining a
# copy of this software and associated documentation files (the "Software"),
# to deal in the Software without restriction, including without limitation
# the rights to use, copy, modify, merge, publish, distribute, sublicense,
# and/or sell copies of the Software, and to permit persons to whom the
# Software is furnished to do so, subject to the following conditions:
#
# The above copyright notice and this permission notice (including the next
# paragraph) shall be included in all copies or substantial portions of the
# Software.
#
# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
# DEALINGS IN THE SOFTWARE.
#
###############################################################################

Set paths used in config files for finding X utilities to look in /usr/bin
even though --bindir is set to /usr/sbin

--- a/man/xdm.man	2020-09-21 21:25:02.287276245 -0700
+++ b/man/xdm.man	2020-09-21 21:26:16.692785820 -0700
@@ -415,7 +415,7 @@
 .IP "\fBDisplayManager.\fP\fIDISPLAY\fP\fB.xrdb\fP"
 Specifies the program used to load the resources.  By default,
 .I xdm
-uses \fI BINDIR/xrdb\fP.
+uses \fI /usr/bin/xrdb\fP.
 .IP "\fBDisplayManager.\fP\fIDISPLAY\fP\fB.setup\fP"
 This specifies a program which is run (as root) before offering the
 Login window.  This may be used to change the appearance of the screen
@@ -431,7 +431,7 @@
 See the section \fBStartup Program.\fP
 .IP "\fBDisplayManager.\fP\fIDISPLAY\fP\fB.session\fP"
 This specifies the session to be executed (not running as root).
-By default, \fI BINDIR/xterm\fP is
+By default, \fI /usr/bin/xterm\fP is
 run.  The conventional name is \fIXsession\fP.
 See the section
 .B "Session Program."
@@ -516,7 +516,7 @@
 will fall back to this program.  This program is executed with no
 arguments, but executes using the same environment variables as
 the session would have had (see the section \fBSession Program\fP).
-By default, \fI BINDIR/xterm\fP is used.
+By default, \fI /usr/bin/xterm\fP is used.
 .IP "\fBDisplayManager.\fP\fIDISPLAY\fP\fB.grabServer\fP"
 .IP "\fBDisplayManager.\fP\fIDISPLAY\fP\fB.grabTimeout\fP"
 To improve security,
@@ -823,7 +823,7 @@
 be:
 .nf
 
- :0 Digital-QV local BINDIR/X :0
+ :0 Digital-QV local /usr/bin/X :0
 
 .fi
 The display types are:
@@ -836,8 +836,8 @@
 The display name must be something that can be passed in the \fB\-display\fP
 option to an X program.  This string is used to generate the display-specific
 resource names, so be careful to match the
-names (e.g., use ``:0 Sun-CG3 local BINDIR/X :0'' instead of
-``localhost:0 Sun-CG3 local BINDIR/X :0''
+names (e.g., use ``:0 Sun-CG3 local /usr/bin/X :0'' instead of
+``localhost:0 Sun-CG3 local /usr/bin/X :0''
 if your other resources are specified as
 ``DisplayManager._0.session'').  The display class portion is also used in the
 display-specific resources, as the class of the resource.  This is
@@ -1395,7 +1395,7 @@
 options or other suitable daemon by specifying the server on the command
 line:
 .nf
-xdm \-server \(lq:0 SUN-3/60CG4 local BINDIR/X :0\(rq
+xdm \-server \(lq:0 SUN-3/60CG4 local /usr/bin/X :0\(rq
 .fi
 .PP
 Or, you might have a file server and a collection of X terminals.  The
@@ -1431,13 +1431,13 @@
 .I CHOOSERPATH
 the default chooser
 .TP 20
-.I BINDIR/xrdb
+.I /usr/bin/xrdb
 the default resource database loader
 .TP 20
-.I BINDIR/X
+.I /usr/bin/X
 the default server
 .TP 20
-.I BINDIR/xterm
+.I /usr/bin/xterm
 the default session program and failsafe client
 .TP 20
 .I XDMXAUTHDIR/A<display>\-<suffix>
--- a/config/Makefile.am	2020-09-22 12:56:20.246070279 -0700
+++ b/config/Makefile.am	2020-09-22 12:58:14.541214699 -0700
@@ -65,7 +65,7 @@
 WTMP_SED_FLAGS  = -e 's|WTMP_FILE||' -e 's|WTMP_FLAG||'
 endif
 
-SED_FILES_FLAGS  = -e 's|BINDIR|$(bindir)|'
+SED_FILES_FLAGS  = -e 's|BINDIR|/usr/bin|'
 SED_FILES_FLAGS += -e 's|DEFAULTVT|$(DEFAULTVT)|'
 SED_FILES_FLAGS += $(UTMP_SED_FLAGS) $(WTMP_SED_FLAGS)
 SED_FILES_FLAGS += -e 's|XDMDIR|$(XDMLIBDIR)|'
